designer/server/src/config.ts

```diff
@@ -88,15 +93,18 @@ const schema = joi.object<Config>({
 const result = schema.validate(
   {
     port: process.env.PORT,
-    env: process.env.NODE_ENV,
+    env: process.env.DEPLOYMENT_ENV,
+    nodeEnv: process.env.NODE_ENV,
     managerUrl: process.env.MANAGER_URL,
     submissionUrl: process.env.SUBMISSION_URL,
     previewUrl: process.env.PREVIEW_URL,
     serviceName: 'Submit a form to Defra',
     logLevel: process.env.LOG_LEVEL,
     phase: process.env.PHASE,
-    isProduction: process.env.NODE_ENV === 'production',
-    isDevelopment: !['production', 'test'].includes(`${process.env.NODE_ENV}`),
+    isProduction: process.env.DEPLOYMENT_ENV === 'production',
+    isDevelopment: !['production', 'test'].includes(
+      `${process.env.DEPLOYMENT_ENV}`
+    ),
```

Comment:
When `DEPLOYMENT_ENV` is missing do we definitely want to run `.includes(undefined)`?

Might need to do this?

```suggestion
    isDevelopment: !process.env.DEPLOYMENT_ENV || !['production', 'test'].includes(
      process.env.DEPLOYMENT_ENV
    ),
```

<br>

If you want to keep code size down, can do an all-in-one destructure at the top?

```js
const {
  DEPLOYMENT_ENV,
  LOG_LEVEL,
  NODE_ENV,
  PHASE,
  SESSION_COOKIE_PASSWORD,
  SESSION_TTL,
  // … more
} = process.env
```